Makefile

```
@@ -0,0 +1,37 @@
+.PHONY: help install build typecheck lint test clean pack-check run
+
+help:
```

checkmake (0.3.2)

[warning] Target body for "help" exceeds allowed length of 5 lines (10).

(maxbodylength)